### ISSUE TITLE
Unit3: 5.9 audio files

### ISSUE THREAD

**Issue by jamesthcheung** [MUTOR-2/ScienceOfMusic#5]
unit3_larry1.wav and unit3_Psycho1.wav are probably copyrighted

**Comment by maccallum** [comment]
The Psycho clip is only 1 second long, which I believe must fall within fair use rules. The other is 15 seconds, so I'm not sure, but it's also probably fine, or could be cut down if needed. Another question for @fuxiao1208

**Comment by fuxiao1208** [comment]
Should we ask @psycheloui if she know about these Audios or not? If not, i will try redo two sound files

**Comment by maccallum** [comment]
I made those files :) But I don't know where I found them--the first is from the movie "The Big Lebowski", and the second is from "Psycho". No need to ask Psyche.

**Comment by fuxiao1208** [comment]
hahahaha, ok 👍

**Comment by fuxiao1208** [comment]
i will check it, and let you know!

**Comment by fuxiao1208** [comment]
yes, we can use "Psycho" and the "The Big Lebowski". But for the movie we need write the name and the year of the movie, like "The Big Lebowski (1998)"  :)
